x/market/util/conv/doc.go

```diff
@@ -0,0 +1 @@
+package conv
```

Comment:
What is this file?

Reply:
The ``doc.go`` file is default file in [Cosmos SDK 0.43](https://github.com/cosmos/cosmos-sdk/blob/3f8596c1955e40ef30e4abcd06f2237d132db3a9/internal/conv/doc.go), but I think it will be not use in our project so I will remove this file. Thank you for your feedback.